src/Uno.UI/UI/Xaml/Media/Animation/Timeline.animation.macOS.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.UI.Xaml.Media.Animation
+{
+	partial class Timeline
+	{
+		partial class AnimationImplementation<T>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['T' is not used in the class.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5301553)